projects/ccf-rui/src/app/shared/components/visibility-menu/visibility-menu.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="item-entry" [class.selected]="selection === item" *ngFor="let item of items" (mouseover)="mouseOver(item)" (mouseout)="mouseOut(item)">
```

Comment:
`*ngFor="let item of items; trackBy: getId"` and add a getId funciton on the class.